runtime/primitives/src/currency_id.rs

```diff
@@ -33,3 +37,29 @@ impl Default for CurrencyId {
 		Self::Token(TokenSymbol::MA)
 	}
 }
+
+#[allow(dead_code)]
```

Comment:
Why do we need ``#[allow(dead_code)]``

Reply:
Compiler always reports we don't use it in anywhere.